UICatalog/UICatalog.cs

```diff
@@ -331,6 +332,22 @@ private static Scenario GetScenarioToRun ()
 
 			return menuItems.ToArray ();
 		}
+		private static MenuItem[] CreateKeybindings()
+		{
+
+			List<MenuItem> menuItems = new List<MenuItem> ();
+			var item = new MenuItem ();
+			item.Title = "Keybindings";
+			item.Action += () => {
+				var dlg = new KeyBindingsDialog ();
+				Application.Run (dlg);
```

Comment:
If I use this dialog to change `QuitTopLevel` to something other than CTRL-Q, UI Catalog doesn't update it's UI.

Wouldn't it be good to implement that as test that all this works end to end?